webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -118,7 +118,7 @@ const AuditCard = ({ hostName }) => {
       </Accordion>
       <Button
         component="a"
-        href={foremanUrl(`/audits?seach=host+%3D+${hostName}`)}
+        href={foremanUrl(`/audits?search=host+%3D+${hostName}`)}
```

Comment:
This looks like an actual bug that should be cherry picked.

Reply:
I agree, though I think it's still experimental page (?)
cc @amirfefer

Reply:
Yes, it is an experimental page